utility_network/trace-a-utility-network/src/main/java/module-info.java

```diff
@@ -0,0 +1,9 @@
+module trace.a.utility.network.main {
+  requires com.esri.arcgisruntime;
+  requires javafx.base;
+  requires javafx.controls;
+  requires javafx.fxml;
+  requires javafx.graphics;
```

Comment:
```suggestion
  // require ArcGIS Runtime module
  requires com.esri.arcgisruntime;
  
  // require JavaFX modules
  requires javafx.base;
  requires javafx.controls;
  requires javafx.fxml;
  requires javafx.graphics;
```
Further to what Colin said below about separating requires/exports, let's have a couple of comments here and add space between Runtime and JavaFX, to emphasise what a developer has to do to include Runtime at a quick glance. Could you apply these changes to the other PR too as per my comment above. Thanks.